elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -182,6 +176,18 @@ def find_bugs_sweep(runtime: Runtime, advisory_id, default_advisory_type, check_
                     f"defined in the assembly config: {excluded_bug_ids}")
         bugs = [bug for bug in bugs if bug.id not in excluded_bug_ids]
 
+    return bugs
+
+
+def find_bugs_sweep(runtime: Runtime, advisory_id, default_advisory_type, check_builds, major_version, find_bugs_obj,
+                    output, brew_event, noop, count_advisory_attach_flags, bug_tracker):
+    if output == 'text':
+        statuses = sorted(find_bugs_obj.status)
+        tr = bug_tracker.target_release()
+        green_prefix(f"Searching {bug_tracker.type} for bugs with status {statuses} and target releases: {tr}\n")
```

Comment:
Could this be `logger.info`?

Reply:
Inclined to keep this as is, we can discuss in https://github.com/openshift/elliott/pull/424 which is touching this also